lib/recurly/schema.rb

```diff
@@ -42,11 +42,11 @@ def self.get_recurly_class(type)
       raise ArgumentError, "#{type.inspect} must be a symbol but is a #{type.class}" unless type.is_a?(Symbol)
 
       if type == :Address
-        Resources::Address
-      elsif Requests.const_defined?(type)
-        Requests.const_get(type)
-      elsif Recurly::Resources.const_defined?(type)
-        Resources.const_get(type)
+        Recurly::Resources::Address
+      elsif Recurly::Requests.const_defined?(type, false)
```

Comment:
Do we need to do the same to this call? https://github.com/recurly/recurly-client-ruby/blob/65f65171cc2cbb6db99f2ea4c8b327ae12577fcc/lib/recurly/schema/json_parser.rb#L64

Reply:
Done